windows/src/engine/kmcomapi/com/options/keymanoptions.pas

```diff
@@ -119,10 +120,10 @@ procedure TKeymanOptions.Apply;
     begin
       if ValueExists(SRegValue_UnderlyingLayout)
         then FOldBaseLayout := StrToIntDef('$'+ReadString(SRegValue_UnderlyingLayout),0)   // I3759
-        else FOldBaseLayout := GetDefaultKeyboardID;
+        else FOldBaseLayout := TBaseKeyboard.GetDefaultKeyboardID;
     end
     else
-      FOldBaseLayout := GetDefaultKeyboardID;
+      FOldBaseLayout := TBaseKeyboard.GetDefaultKeyboardID;
```

Comment:
See, I'd like to point out that here, we're using "base layout" nomenclature for the variable that receives the returned value.  Consistency on this point would also be nice.

(In reference to the other code-block comment of mine.)